modules/renter/hostdb/hostdb.go

```diff
@@ -58,5 +118,7 @@ func New() *HostDB {
 	}
 	go hdb.threadedScan()
 
-	return hdb
+	cs.ConsensusSetSubscribe(hdb)
```

Comment:
Now that the consensus set has persistent subscriptions, we can use those and it shouldn't affect startup time.